e2e-tests/web/wdio.conf.ts

```diff
@@ -1,36 +1,30 @@
+import { localCapabilities } from './capabilities'
+
 export const config = {
   runner: 'local',
   specs: ['./web/test/specs/**/*.ts'],
   exclude: [],
   maxInstances: 10,
 
-  capabilities: [
-    {
-      maxInstances: 5,
-      browserName: 'chrome'
-    }
-  ],
+  capabilities: [process.env.CI ? localCapabilities.ci : localCapabilities.chrome],
   logLevel: 'info',
   coloredLogs: true,
   bail: 0,
   baseUrl: 'http://localhost:9000',
   waitforTimeout: 100000,
   connectionRetryTimeout: 120000,
   connectionRetryCount: 3,
-  services: ['selenium-standalone'],
+  services: process.env.CI ? [] : ['selenium-standalone'],
   framework: 'jasmine',
   reporters: ['junit'],
 
-  jasmineNodeOpts: {
-    defaultTimeoutInterval: 120000
+  jasmineOpts: {
+    defaultTimeoutInterval: 50000
   },
 
-  onPrepare: async function (): Promise<void> {
-    const startupDelay = 20000
-    await new Promise(resolve => setTimeout(resolve, startupDelay))
-  },
+
 
   before: async function (): Promise<void> {
-    await browser.setTimeout({ implicit: 80000 })
+    await browser.setTimeout({ implicit: 80000, pageLoad: 40000 })
```

Comment:
Did you add this because of webpack startup times?

Reply:
yep. Will do the same thing as the onPrepare hook. And and also increases the general page loading time limits

Reply:
Correction: I readded the initial startup delay, as the webapp takes about 5 seconds until the packager started the server. I still think the pageLoad is worth it to avoid failing tests due to web app instability